src/api.py

```diff
@@ -0,0 +1,26 @@
+from flask import Flask, request
+from src import service
+
+app = Flask(__name__)
+
+
+# Sample url: https://tmt.org/?url=https://github.com/teemtee/tmt&name=/tests/core/smoke
+# or for plans: https://tmt.org/?url=https://github.com/teemtee/tmt&name=/plans/features/basic&type=plan
+@app.route("/", methods=["GET"])
+def find_test():
+    test_url = request.args.get("test_url", default=None)
```

Comment:
In `tmt` we're consistently using dash `-` in multiword keys. Let's keep the naming consistent here as well and use `test-name`, `test-ref`...